docs/usage/types.md

```diff
@@ -828,16 +832,16 @@ The following arguments are available when using the `conbytes` type function
 
 ## Strict Types
 
-You can use the `StrictStr`, `StrictBytes`, `StrictInt`, `StrictFloat`, and `StrictBool` types 
+You can use the `StrictStr`, `StrictBytes`, `StrictInt`, `StrictFloat`, and `StrictBool` types
 to prevent coercion from compatible types.
 These types will only pass validation when the validated value is of the respective type or is a subtype of that type.
-This behavior is also exposed via the `strict` field of the `ConstrainedStr`, `ConstrainedBytes`, 
+This behavior is also exposed via the `strict` field of the `ConstrainedStr`, `ConstrainedBytes`,
 `ConstrainedFloat` and `ConstrainedInt` classes and can be combined with a multitude of complex validation rules.
 
 The following caveats apply:
 
 - `StrictBytes` (and the `strict` option of `ConstrainedBytes`) will accept both `bytes`,
-   and `bytearray` types. 
+   and `bytearray` types.
```

Comment:
Are the line-endings changing here?

Reply:
As mentioned in my [last comment](https://github.com/samuelcolvin/pydantic/pull/2434#issuecomment-813412245) trailing spaces were automatically deleted by editor. Since those whitespaces were followed by a single newline (which markdown->html should render as a space), the resulting HTML _should_ be identical. 

As I mentioned, should those trailing spaces were left on purpose, I can revert the change.

Reply:
No problem. Just wanted to double check.

I just opened a PR that should prevent developers from experiencing the same issue in the future (without having to mess with your git or editor settings).